src/lib/runtime_config/runtime_config.ml

```diff
@@ -1362,24 +1362,18 @@ let make_fork_config ~staged_ledger ~global_slot ~blockchain_length
             { ledger = ledger_of_accounts accounts; seed = next_epoch_seed } )
     }
   in
-  let update =
-    make
-    (* add_genesis_winner must be set to false, because this
-       config effectively creates a continuation of the current
-       blockchain state and therefore the genesis ledger already
-       contains the winner of the previous block. No need to
-       artificially add it. In fact, it wouldn't work at all,
-       because the new node would try to create this account at
-       startup, even though it already exists, leading to an error.*)
-      ~epoch_data ~genesis
-      ~ledger:
```

Comment:
The `genesis` argument is now gone, but we're still computing that argument above. I've fixed it properly [here](https://github.com/MinaProtocol/mina/pull/15109/files) anyway, but try not to leave dangling unused logic in the code in future.